src/service/router.ts

```diff
@@ -6,16 +6,20 @@ import {
 import express from 'express';
 import Router from 'express-promise-router';
 import { Logger } from 'winston';
+import { CatalogClient } from '@backstage/catalog-client';
+import { FetchApi } from '@backstage/core-plugin-api';
 import { BlamelessJob } from './cron-job';
 
 export interface RouterOptions {
   logger: Logger;
+  catalogClient: CatalogClient;
```

Comment:
Do you need this here? It seems like you arent using it and instantiating the catalog client yourself?

Reply:
Yes that's right, since i am instatiating it directly, we don't need it here.